pdr-admin/src/app/app-routing.module.ts

```diff
@@ -68,6 +60,22 @@ const routes: Routes = [
       icon: 'bi-clock-history',
     },
   },
+  {
+    path: 'protected-area/:id',
```

Comment:
thought: should probably sync up our definitions for park/protected area between API and FE (API is currently `/parks/:id?*`). I feel like it would be good boy code to have our API/FE routes match.

Reply:
From what I understand, protected areas and sites are going to diverge in terms of model which is why I've created a protected-area module and site module. Are both sites and protected-areas described as parks?

Reply:
They will diverge - sites are not parks, they are subset assets of them.